lib/install.js

```diff
@@ -0,0 +1,101 @@
+const { resolve } = require('path');
+const fs = require('fs-extra');
+const assert = require('bsert');
+const validate = require('validate-npm-package-name');
+const { format } = require('prettier');
+
+const { checkExistence, npmExists } = require('./helpers');
+
+function filterArgs() {
+  // remove node, path, and command args
+  const args = process.argv.slice(3);
+  let names = args.filter(name => name[0] !== '-');
+  return names;
+}
+
+module.exports = (config, logger) => async (name, cmd) => {
+  let bpanelConfig, CONFIG_PATH;
```

Comment:
Any reason these are declared at the top? Everything happens inside of the `try`

Reply:
good catch, that was left over from a previous iteration.